zeth_contracts/contracts/BW6_761_test.sol

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+pragma solidity ^0.5.0;
+
+contract BW6_761_test
+{
+    // In many cases, these numbers must be used as literals in the assembly
+    // code.
+
+    uint256 constant scalarWords = 2;
+    uint256 constant scalarBytes = scalarWords * 32; // 64 (0x40)
+    uint256 constant coordWords = 3;
+    uint256 constant coordBytes = 3 * 32; // 96 (0x60)
+    uint256 constant pointWords = 2 * coordWords;
+    uint256 constant pointBytes = pointWords * 32; // 192 (0xc0)
+
+    /// `points` should be the concatenation of 2 encoded points
```

Comment:
NatSpec is not followed consistently here. Maybe worth switching to `//` for this test file here

Reply:
switched to `// `